app/uk/gov/hmrc/vatapi/resources/FinancialDataResource.scala

```diff
@@ -40,57 +43,72 @@ class FinancialDataResource @Inject()(
 
   def retrieveLiabilities(vrn: Vrn, params: FinancialDataQueryParams): Action[AnyContent] = APIAction(vrn).async { implicit request =>
 
+    def audit(desResponse: Option[Response], auditResponse: AuditResponse) = {
+      auditService.audit(AuditEvents.retrieveVatLiabilitiesAudit(Response.getCorrelationId(desResponse),
+        request.authContext.affinityGroup, getArn, auditResponse))
+    }
+
     logger.debug(s"[FinancialDataResource][retrieveLiabilities] Retrieving Liabilities from DES")
     val result = fromDes {
       for {
         response <- execute { _ => connector.getFinancialData(vrn, params) }
       } yield response
-    } onSuccess { response =>
-      response.filter {
-        case 200 =>
-          response.getLiabilities(vrn) match {
+    } onSuccess { desResponse =>
```

Comment:
Why has this been renamed? Is it named desResponse in other places?

Reply:
For clarity. Because calling it response is confusing (esp in a controller). It's not our response; it's DES's response. Have made similar changes to the other endpoint in this PR.